### PR TITLE
add tizen-8-0-m1.md release note for public release 

### PR DESCRIPTION
**Pull request by seokwon-jung**
### Change Description ###

add tizen-8-0-m1.md for public release

### CODE REVIEW

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+- Package Manager
+  - New feature has been ready to remove user data from other applications.
+- App-control 
+  - Launch-bounds APIs have been provided.
+  - Extended appid for multi-instance has been introduced.
+- TIDL
+  - The TIDL compiler has been updated to generate code that use Cion API for Dart language
+- Tizen Database Connectivity (TDBC)
+  - New feature has been ready to access  a database.
+    - Provides an C# API which defines how a client may access a database
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:156]:
```suggestion
    - Provides an C# API which defines how a client may access a database.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+- Vulkan
+  - The version of SPIRV-Cross, SPIRV-Headers and SPIRV-Tools has been upgraded to 1.3.239.
+  - The version of Vulkan-Headers, Vulkan-Hpp, Vukan-Loader and Vulkan-ValidationLayers has been upgraded to 1.3.240.
+  - The version of glslang has been upgraded to 1.3.239.
+- Libinput
+  - The version of Libinput has been upgraded to 1.22.0.
+- Libxkbcommon
+  - The version of Libxkbcommon has been upgraded to 1.5.1.
+- TTS Framework
+  - Supports root daemon as TTS client
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:208]:
```suggestion
  - Supports root daemon as TTS client.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+  - Bonded devices response not coming issue has been fixed.
+  - Infinite callback issue in EOF case has been fixed.
+
+
+### Security
+
+#### New and changed features
+
+- Privacy privilege manager
+  - Privacy privilege manager feature (PPM) has been deprecated.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:352]:
```suggestion
  - Privacy privilege manager (PPM) feature has been deprecated.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+### Web framework
+
+#### New and changed features
+
+- Web Engine
+  - Open source chromium 108 version base has been applied.
+  - OzonePlatform for EFL has been added.
+  - Ozone EFL based Onscreen/Offscreen rendering mode has been supported.
+  - Offscreen rendering mode based NUI feature has been supported.
+  - ESPP based Media Renderer has been added.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:374]:
```suggestion
  - ESPP based media renderer has been added.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 5]
```diff
@@ -0,0 +1,506 @@
...
+
+#### New and changed features
+
+- Lightweight Web Engine
+- Web Engine
+- Service Worker lifecycle has been supported.
+- Caching responses on network requests in the Service Worker has been supported.
+- Maplike declaration has been added.
+- JavaScript Engine
+- Ecmascript feature (from es2022) has been added.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:394]:
```suggestion
- ECMAscript feature (from ES2022) has been added.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 6]
```diff
@@ -0,0 +1,506 @@
...
+  - Tizen.Content.Download
+    - New caching APIs have been added.
+
+
+### Toolchain
+
+#### New and changed features
+
+- GCC
+  - arm target changes has been back-ported from releases/gcc-9.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:438]:
```suggestion
  - ARM target changes have been back-ported from releases/gcc-9.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 7]
```diff
@@ -0,0 +1,506 @@
...
+  - Libsanitizer: AddressSanitizer overhead reduction option has been added.
+- Glibc
+  - Timezone: updated from tzcode 2020a
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:441]:
```suggestion
  - Timezone: updated from tzcode 2020a.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 8]
```diff
@@ -0,0 +1,506 @@
...
+- Binutils
+  - Packaging: keep shared libraries for libbfd & libopcodes.
+
+#### Fixes
+
+- GCC
+  - Bugfix PR c++/61414 patchset has been applied.
+  - Bugfix PR middle-end/98189, 95886 patchset has been applied.
+- Glibc
+  - Static TLS(Thread Local Storage) block memory allocation bugfix patchset has been applied.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:451]:
```suggestion
  - Static TLS (Thread Local Storage) block memory allocation bugfix patchset has been applied.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 9]
```diff
@@ -0,0 +1,506 @@
...
+
+#### New and changed features
+
+- Machine Learning (ML) API updates
+  - ML.Service API Updates
+    - Add new APIs to separately manage ML model files. These APIs allow ML applications to exploit the latest ML model deployed without code-level modifications that require re-packaging and re-distribution of the whole application.
+    - AI Inference offloading between Tizen and TizenRT has been improved. The binary size of TizenRT is optimized (< 100kB). 
+  - ML.Inference API Updates
+    - Add new API to use extended rank limit. To maintain the backward compatibility, the default rank limit is 4. The extended rank is only used when new API is called.
+    - Support TensorFlow-Lite v2.11.0 by default.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:474]:
```suggestion
    - Support TensorFlow Lite v2.11.0 by default.

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 10]
```diff
@@ -0,0 +1,506 @@
...
+  - ML.Training API Updates
+    - Add Identify Layer as a utility layer which flows everthing as it is.
+    - Add Learning Rate Scheduling Features.
+- Step Learning Rate Scheduling
+- Exponent Learning Rate Scheduling
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:479]:
```suggestion
    - Add learning rate scheduling features.
- Step learning rate scheduling
- Exponent learning rate scheduling

```

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 11]
```diff
@@ -0,0 +1,506 @@
...
+    - Model management feature such as register, fetch active model, update, and delete has been added. Using this feature, ML applications can share their model with other applications.
+  - Pipeline based data repository and training feature has been added.
+    - New NNStreamer elements such as datareposrc, datareposink, and tensor_trainer have been added.
+    - Support for the training in AI pipeline has been added.
+- NNTrainer updates
+  - Support proactive swap which utilize secondary storage for less memory consumption
+    - Add Cache Pool / Cache Loader / Cache Element classes 
+    - Update Memory pool and planner for better utilization
+  - Added Execution order and memory usage tarcing for debugging
+  - Added TFLite exporter which generate inference model for tflite (unstable)
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-8-0-m1.md:496]:
```suggestion
  - Support proactive swap which utilizes secondary storage for less memory consumption.
    - Add Cache Pool/Cache Loader/Cache Element classes.
    - Update Memory pool and planner for better utilization.
  - Added Execution order and memory usage tarcing for debugging.
  - Added TFLite exporter which generates inference model for tflite (unstable).

```

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Hello. Hope you are well.

I have gone through the release note and have provided some suggestions. Please check and implement them.

File: docs/platform/release-notes/tizen-8-0-m1.md [hunk 12]
```diff
@@ -0,0 +1,506 @@
...
+  - Display rotation function has been supported.
+  - Power suspending and resuming callback function have been supported.
+  - Power suspending cancellation function has been supported.
+  - Power lock status callback has been supported.
+  - Power wakeup reason function has been supported.
+  - Uart flush and drain function have been supported.
+  - Sensor listener attribute getter function has been supported.
+  - Sensor attribute setter/getter functions have been supported.
+  - Sensitivity attribute of Proximity sensor has been supported.
+  - Sensord signal handling has been improved by removing race conditions.
```
[the remark this answers is not in the document]
Comment (seokwon-jung) [reply docs/platform/release-notes/tizen-8-0-m1.md:29]: Hello~, actually sensord is specific module name(proper name) which means "sensor-daemon"
you can search easily by googling also.
what should I do? please let me know :-)

**Comment by seokwon-jung** [comment]
@safir-srbd ,hello
I've updated it as your recommendation.
please check once more :-)
thx

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Thank you for making the suggested changes. However, it seems like you missed a few. I have marked them down.


**IMPORTANT**: Also, you have to add this file to the `toc_all.md` file in the Platform folder. Otherwise, the release note will not appear on the Left Navigation Bar.

**Comment by seokwon-jung** [comment]
> @seokwon-jung Thank you for making the suggested changes. However, it seems like you missed a few. I have marked them down.
> 
> **IMPORTANT**: Also, you have to add this file to the `toc_all.md` file in the Platform folder. Otherwise, the release note will not appear on the Left Navigation Bar.
hello~, below commit is fine?
[add 8.0 m1 to toc_all.md](https://github.com/Samsung/tizen-docs/pull/1824/commits/34decbc03885519567e485073eb8568d0c30cbee)
[34decbc](https://github.com/Samsung/tizen-docs/pull/1824/commits/34decbc03885519567e485073eb8568d0c30cbee)

**Review by safir-srbd — APPROVED** [review]
@seokwon-jung After going through the document one more time, everything seems to be in order. I am approving this PR for merging, however I'll wait to merge this till before the release date in case anything needs to be changed or added.

**Review by safir-srbd — APPROVED** [review]
@seokwon-jung After going through the document one more time, everything seems to be in order. I am approving this PR for merging, however I'll wait to merge this till before the release date in case anything needs to be changed or added.

**Pull request merged**